pipeline-runner/R/qc-7-embed_and_cluster.R

```diff
@@ -353,7 +364,7 @@ detect_variable_types <- function(cl_metadata) {
   # remove samples var, useless from this point on
   clm_cols <- grep("^samples$", clm_cols, value = T, invert = T)
 
-  clm_cols <- c(clm_cols, "duplicate_barcode")
+  clm_cols <- union(clm_cols, "duplicate_barcode")
```

Comment:
Add comment explaining that duplicate_barcode should always be type CLM. However, it might have been detected as CLMPerSample so we need to set it manually. Also remove it manually from the CLMPerSample in the case that it is)